VMEncryption/main/handle.py

```diff
@@ -927,8 +927,7 @@ def enable_encryption():
                                           code=str(CommonVariables.configuration_error),
                                           message='Failed to encrypt resource disk. Please make sure no process is using it.')
                         else:
-                            logger.log("Resource disk encrypted successfully.")
-
+                            logger.log("Resource disk is not present or encrypted successfully.")
```

Comment:
Ambiguous sentence.
Change to "Resource Disk is either absent or encrypted successfully."